plugins/catalog-react/src/alpha/blueprints/EntityContentBlueprint.ts

```diff
@@ -91,5 +99,9 @@ export const EntityContentBlueprint = createExtensionBlueprint({
     } else if (typeof filter === 'function') {
       yield entityFilterFunctionDataRef(filter);
     }
+
+    if (group) {
+      yield entityContentGroupDataRef(group);
```

Comment:
Note for followup: might be that we want to simply skip yielding if `group === false` rather than including `false` as a potential value of the data.

Reply:
Followup in https://github.com/backstage/backstage/pull/28917